### PR TITLE
feat: initial validation

### LINKED ISSUE #22: VALIDATION: Responsibly Validate all Models

### PR DESCRIPTION
**Pull request by miggitymack**
closes #22

### CODE REVIEW

File: db/models/group.js [hunk 1]
```diff
@@ -4,7 +4,10 @@ const { STRING } = require('sequelize');
 
 module.exports = db =>
   db.define('groups', {
-    name: STRING,
+    name: {
+      type: STRING,
+      allowNull: false
```
Comment (dffattal) [inline db/models/group.js]: We need a
```
validate: {
    notEmpty: true
}
```
as well as allowNull anytime we don't want users to submit an empty string.

File: db/models/task.js [hunk 2]
```diff
@@ -4,12 +4,20 @@ const { STRING } = require('sequelize');
...
 module.exports = db =>
   db.define('tasks', {
-    description: STRING,
-    status: STRING
+    description: {
+      type: STRING,
+      allowNull: false
+    },
+    status: { type: STRING
+      defaultValue: 'pending'
```
Comment (dffattal) [inline db/models/task.js]: Can you change this to 'Pending'? I think some stuff on the frontend checks for the capital P.

**Review by JeffKandel — DISMISSED** [review]
Merging so things are up to date.

**Comment by JeffKandel** [comment]
Just kidding.  Not merging.  The seed file doesn't run.

**Review by dffattal — APPROVED** [review]
We may need some more validations later on but this is good enough for now.

**Pull request merged**